docs/manual/getting_started.rst

```diff
@@ -2046,24 +2231,23 @@ stored in the HelloWorldData.idl file (same as in chapter 3).
         };
     };
 
-A subset of the OMG Interface Definition Language (IDL) is used as DDS
+The OMG Interface Definition Language (IDL) subset uses a DDS
 data definition language. In our simple example, the HelloWorld data
```

Comment:
I think this change incorrectly conveys the meaning of the original sentence. Perhaps:
```suggestion
The data definition language used for DDS corresponds to a subset of 
the OMG Interface Definition Language (IDL). In our simple example, the HelloWorld data
```